src/main/java/org/jalasoft/moi/model/main.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2020 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+*/
+
+package org.jalasoft.moi.model;
+
+public class main {
+    public static void main(String[] arg){
+        String comm1="cmd.exe /c \"cd C:/Users/MauricioOroza/com && javac *.java && java MainClass\"";
```

Comment:
I believe that the `main` class is good for try that your code it's work, but for the project this will not needed, Let's use unit tests as an alternative to try the code.